ray-curator/ray_curator/utils/nvcodec_utils.py

```diff
@@ -0,0 +1,472 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""nvcodec_utils.
+
+Various helpers for GPU accelerated decode/encode.
+"""
```

Comment:
This file is great. NITs

1. Can we ensure that what @thomasdhc is doing does not cause any of these imports to fail.
2. Can we move this to ray_curator/video/utils/....py, maybe not now but in a later PR.